Dockerfile

```diff
@@ -32,16 +32,16 @@ RUN \
         yum-config-manager --enable osg-${BASE_YUM_REPO}; \
     fi && \
     yum clean all && \
-    rm -rf /var/cache/yum/
+    rm -rf /var/cache/yum/ && \
+    # Impatiently ignore the Yum mirrors
+    sed -i 's/\#baseurl/baseurl/; s/mirrorlist/\#mirrorlist/' \
```

Comment:
Do the `\` line continuations work across comment lines?

Reply:
Yup, this seems to work! This is a good reminder, though that we should apply these sed commands to all OSG repos